src/driver/RealtimeDatabase/InProcessRealtimeDatabase.ts

```diff
@@ -356,7 +360,7 @@ export class InProcessRealtimeDatabaseRef
         eventType: "value",
     ): Promise<InProcessFirebaseRealtimeDatabaseSnapshot> {
         if (eventType !== "value") {
-            throw new Error('Only the "value" event type is supported.')
+            throw new Error("Only the \"value\" event type is supported.")
```

Comment:
What's the reason for this change? seems like it was better before

Reply:
Ah this is just automatic prettier/eslint, will revert